.changeset/popular-plums-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": minor
+---
+
+Add a Lucide Icons as dependency with a specific subset available
```

Comment:
We could be a little more descriptive in this changeset and list the new icon options available.